new-frontiers-14/frontier-station-14
# Kegs + Urn Update

## About the PR
Fixed kegs and urn paper slots, added new option to close and open kegs.

## Why / Balance
We got new comps to play with and we didnt add them.

## How to test
Spawn kegs
Add note to it, inspect
Open it, yeet to see it spill

Spawn urn, add a note and inspect it.

## Media

## Breaking changes
N/A

**Changelog**
:cl: dvir01
- tweak: Kegs can now be closed for safe transfer, and the note on them is much more easy to read.
- tweak: Urn note is much more easy to read.
